db/migrate/20210524151343_create_foods.rb

```diff
@@ -0,0 +1,11 @@
+class CreateFoods < ActiveRecord::Migration[5.2]
```

Comment:
No need for this migration/table. Our poros can handle this, and we don't want to store info from the API unless there is no way around that.